unfinished/openai/__ini__.py

```diff
@@ -54,31 +53,36 @@ def session_auth(self):
             'user-agent': 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/112.0.0.0 Safari/537.36',
         }
 
-        return self.client.get('https://chat.openai.com/api/auth/session', headers=headers).json()
+        response = self.session.get('https://chat.openai.com/api/auth/session', headers=headers)
+        return response.json()
 
-    def send_message(self, message):
-        response = self.client.post('https://chat.openai.com/backend-api/conversation', json={
-            'action': 'next',
-            'messages': [
-                {
-                    'id': str(uuid4()),
-                    'author': {
-                        'role': 'user',
-                    },
-                    'content': {
-                        'content_type': 'text',
-                        'parts': [
-                            message,
-                        ],
-                    },
+def send_message(self, message):
+    # Define the request payload
```

Comment:
Can you remove the comment please? The code is self explanatory no need to comment it.